server/src/controllers/configController.js

```diff
@@ -1,11 +1,24 @@
 import Config from '../models/config.js';
 
 export function createOne(request, response) {
-  const newConfig = new Config({});
+  const newConfig = new Config({
+    "data.serverType": request.body.data.serverType,
+    "data.appName": request.body.data.appName,
+    "data.serverSettings.port": request.body.data.serverSettings.port,
+    "data.routers": [{
+      id: request.body.data.routers[0].id,
+      startPoint: request.body.data.routers[0].startPoint,
+      endPoints: request.body.data.routers[0].endPoints,
```

Comment:
Same thing here, `endPoints` should be standardized, and I'm not sure what it is in the body. If it is being sent through the request as that, we should change that as well.